src/constants.ts

```diff
@@ -8,3 +8,16 @@ export const ChainSlugToProject: { [chainSlug in ChainSlug]?: Project } = {
   [ChainSlug.LYRA]: Project.LYRA,
   [ChainSlug.SX_NETWORK_TESTNET]: Project.SX_NETWORK_TESTNET,
 };
+
+// pool Id is used to identify a source when multiple tokens use same controller. This is useful for the cases like merging inbounds
+// for USDC and USDCE so we dont have 2 versions of USDC on app chain.
+export const tokensWithOnePoolId: Tokens[] = [Tokens.USDCE, Tokens.WETH];
+
+export const tokensAllowingMergingInbound: {
```

Comment:
this not used as of now, but can be used if we have more complex token relationships

Reply:
okay 👍🏼